tapdb/assets_store.go

```diff
@@ -3542,6 +3542,13 @@ func (a *AssetStore) QueryParcels(ctx context.Context,
 					"anchor tx: %w", err)
 			}
 
+			// Fill in the anchor transaction's output pkScripts.
+			for i, out := range outputs {
+				outIdx := out.Anchor.OutPoint.Index
+				pkScript := anchorTx.TxOut[outIdx].PkScript
+				outputs[i].Anchor.PkScript = pkScript
+			}
```

Comment:
I think some unit tests are panicking on index out of range.

Reply:
Ah yes indeed, the test needed to be updated. PTAL.